app/src/main/java/com/github/libretube/ui/views/CustomExoPlayerView.kt

```diff
@@ -172,17 +172,9 @@ internal class CustomExoPlayerView(
                     )
                 ) {
                     updatePlayPauseButton()
+                    updateScreenOnState()
```

Comment:
Why creating a new function here instead of running the code directly?
The function only contains one line and is only called once, so that seems like a bad practice.

Reply:
I put a single line of code in a function to make the code more readable. If it's not a good way to do it, I'll change it and submit the code again for review.